tests/testthat/test-quiet.R

```diff
@@ -95,6 +112,9 @@ test_that("Using quiet supresses messages as expected", {
         })
     })
 
+    skip_if(Sys.getenv('USER') %in% c('secrestm','gowerc','sabanesd'))
```

Comment:
Whats the reason for this sorry, am super not keen on having usernames coded into the package.

Reply:
@gowerc yeah good point. The reason is that `skip_on_cran()` and `skip_on_ci()` were not suppressing the warnings to `devtools::check()` locally and that got annoying.

Reply:
Interesting, I would be keen to understand more before going ahead with such a blunt solution. That being said, assuming we do need such a blunt solution, I would much prefer to use a environment variable flag rather than specific usernames.

i.e.

```
skip_if(  ! Sys.getenv("RUN_ALL_TEST") == TRUE)
```

Reply:
@gowerc yeah, agreed this is a much better approach. Thanks for the suggestion. I personally am okay with your proposed line in `skip_if()` because `devtools::test()` is working. I blame the error to problems updating global options in `devtools::check()`.

Reply:
@gowerc implemented.